client/task-list/index.js

```diff
@@ -124,9 +124,17 @@ export class TaskDashboard extends Component {
 		} = this.props;
 		const { isCartModalOpen } = this.state;
 		const allTasks = this.getAllTasks();
-		const { extension: extensionTasks, setup: setupTasks } = allTasks;
+		const { extension, setup: setupTasks } = allTasks;
 		const { task } = query;
 
+		const extensionTasks = extension.sort( ( a, b ) => {
```

Comment:
I'm not sure if it's intended, but I get `Cannot read property 'sort' of undefined` from here without `add-task` enabled.

Reply:
Strange, I'm not having that error occur locally, but I can reproduce it by modifying the example to return a non-array. 🤷‍♂️

Reply:
Oh whoops - just needed to test without the example plugin activated!

Reply:
Fixed in a45344d.